docs/user-guide/kubectl/kubectl_annotate.md

```diff
@@ -85,6 +85,7 @@ $ kubectl annotate pods foo description-
       --output-version="": Output the formatted object with the given version (default api-version).
       --overwrite[=false]: If true, allow annotations to be overwritten, otherwise reject annotation updates that overwrite existing annotations.
       --resource-version="": If non-empty, the annotation update will only succeed if this is the current resource-version for the object. Only valid when specifying a single resource.
+  -a, --show-all[=false]: When printing, show all resources (default hide terminated ones.)
```

Comment:
Since we have now restricted the filtering to only pods, I am wondering if it is better to say "default hide terminated pods" instead of "default hide terminated ones"